src/types/item.ts

```diff
@@ -3,4 +3,5 @@ export interface Item {
   slot: 1 | 2 | 3 | 4 | 5 | 6 | 7
   shared?: boolean
   id?: string
+  sharePodcast?: []
```

Comment:
```suggestion
  id?: string
```

Reply:
I have removed the `shared` field but `sharePodcast` is used for which item is shared to which podcast so we can't remove this field.
Please refer -> https://i.imgur.com/71e4MRr.png

Reply:
its a copy, theres no value in tracking the content of the share, or suggesting there is some capability to unshare it be unticking it